manageiq-providers-ibm_cloud.gemspec

```
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "ibm_cloud_power", "~> 1.1", ">= 1.1.1"
   spec.add_dependency "ibm_cloud_resource_controller", "~> 2.0"
   spec.add_dependency "ibm-cloud-sdk", "~> 0.1"
-  spec.add_dependency "ibm_cloud_activity_tracker", "~> 0.1"
+  spec.add_dependency "ibm_cloud_activity_tracker", "~> 0.1.1"
```

We want to allow minor version bumps (e.g. 0.1.1 -> 0.2.0) so would like to keep the `~> 0.1` and just add an additional `>= 0.1.1` after
```suggestion
  spec.add_dependency "ibm_cloud_activity_tracker", "~> 0.1", ">= 0.1.1"
```